teachertool/src/global.scss

```diff
@@ -135,6 +135,86 @@ code {
     }
 }
 
+/*******************************/
```

Comment:
I guess the breakpoints in https://github.com/microsoft/pxt/blob/master/react-common/styles/react-common-breakpoints.less are inaccessible to us since it's less, but I wonder if it's worth mirroring that pattern for easy portability between our components and react-common...not something we need to block on, though.

Reply:
Makes sense to mirror the breakpoints for portability.